public/js/test/mochitest/browser_dbg-navigation.js

```diff
@@ -39,4 +38,10 @@ add_task(function* () {
   );
 
   is(countSources(dbg), 4, "4 sources are loaded.");
+
+  // Test that the current select source persists across reloads
+  yield selectSource(dbg, "long.js");
+  yield reload(dbg, "long.js");
+  ok(getSelectedSource(getState()).get("url").includes("long.js"),
```

Comment:
Not sure I'd really care about scroll location here; I don't think we want to persist those across reloads. That seems tricky because the source may have changed a good bit. I don't think Chrome does that, right? We probably want to just save the current tab URLs.

I think I might go ahead and merge as is, and if we do really want to test more stuff we can move this into a helper functions. The pause and highlight locations are nice because there's a bunch of stuff to test, but for other things I'm OK querying the state directly. Other tests will make sure that the current selected source has the right tab and everything.

(Definitely still feeling this out though so we may abstract it out later...)